insights/parsers/nfs_exports.py

```diff
@@ -127,14 +64,20 @@ class NFSExportsBase(Parser):
     """
     Class to parse ``/etc/exports`` and ``/etc/exports.d/*.exports``.
 
+    Exports are stored keyed on the path of the export, and then the host
+    definition.  The flags are stored as a list.  NFS allows the same path
+    to be listed on multiple lines and in multiple files, but an exported
+    path can only have one definition for a given host.
+
     Attributes:
         data (dict): Key is export path, value is a dict, where the key is the
             client host and the value is a list of options.
 
-        ignored_lines (list): List of lines that are not used by ``nfsd``.
+        ignored_exports (dict): A dictionary of exported paths that have host
```

Comment:
Yep, checked that beforehand :-)  Good to confirm though.

Reply:
Changes to existing component interfaces aren't allowed against master. It's becoming impossible for us to check what is being used since we have at least insights-plugins, prodsec, support delivery, and some other teams investigating rules. Feel free to open issues for discussion and submit PRs with these kinds of changes against the dev branch, though.

Reply:
OK, I've added in `ignored_lines` as another reference to `ignored_exports`.